app/javascript/packs/provisioning-actions-common.js

```diff
@@ -0,0 +1,14 @@
+window.provisioningListenToRx = () => {
+  listenToRx ( event => {
+    if (event === undefined || event.type === undefined || event.actionType === undefined) {
+      return;
+    }
+    const { type, actionType, payload: {action, item} } = event;
```

Comment:
this will fail for any messages without a payload

```js
if (!event || !event.payload || event.type !== 'GTL_CLICKED" || event.actionType !== 'provisioning') {
  return;
}

const {action, item} = event.payload;

sendDataWithRx({item: item, type: 'gtlSetOneRowActive'});
￼miqAjax(`${action.url}${item.id}`);
```

Reply:
Yep